Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu:latest
+
+ARG MANIFEST_UTIL_REPO=https://github.com/metrostar/ib_manifest_util.git
+ARG CONDA_VENDOR_REPO=https://github.com/metrostar/conda-vendor.git
+# install micromamba
+RUN apt-get update \
+	&& apt-get install curl bzip2 git -y \
+	&& curl -Ls https://micro.mamba.pm/api/micromamba/linux-64/latest | tar -xvj bin/micromamba
+
+WORKDIR /repos/
+
+# Clone relevant git repos
+RUN git clone $MANIFEST_UTIL_REPO \
+	&& git clone $CONDA_VENDOR_REPO
+
+#copy ib_manifest_util /manifestor/ib_manifest_util
+#
+#copy conda-vendor /manifestor/conda-vendor
+
+# Install micromamba, create env, install correct conda-vendor
+RUN eval "$(micromamba shell hook --shell=dash)" \
```

Comment:
--shell=dash seems wrong

Reply:
Surprisingly enough, that's correct. RUN commands in dockerfiles are run with /bin/sh, which on ubuntu is symlinked to dash. Container builds fail on `micromamba activate` if that's set to bash.